src/Autofac/Builder/BuildCallbackManager.cs

```diff
@@ -11,7 +11,7 @@ namespace Autofac.Builder
     /// </summary>
     internal static class BuildCallbackManager
     {
-        private static readonly TypedService CallbackServiceType = new TypedService(typeof(BuildCallbackService));
+        private static readonly TypedService CallbackServiceType = new(typeof(BuildCallbackService));
```

Comment:
I think this is the sort of `new()` usage that confuses me. I've generally only used `new()` for default constructors, because my brain melts when I try to read it with parameters, but I'll get used to it if you think we should standardise on the pattern.

Reply:
I fixed it so `new()` like this is only in class field sorts of things and not in methods. Agreed, this looks weird. Kinda what I get for "fix all" with the analyzers.